src/core/ts/background-scripts/background.js/lib/BergamotTranslatorAPI.ts

```diff
@@ -66,6 +66,7 @@ interface TranslateRequestWorkerMessage extends WorkerMessage {
 export interface TranslationResults {
   originalTexts: string[];
   translatedTexts: string[];
+  qeAnnotatedTranslatedTexts?: string[];
```

Comment:
What does `?` represent after `qeAnnotatedTranslatedTexts`?

Reply:
It means that the attribute is optional. The WASM API returns no quality estimation metadata, so the attribute will not be set.

Reply:
Thanks @abhi-agg for this question. It reminded me of two important things that were missing in this PR and are now addressed.